README.md

```diff
@@ -197,36 +263,60 @@ Test benchmark code can be found here:
 * [Comparison with jax-libtree](https://github.com/opendilab/treevalue/blob/main/test/compare/jax/test_jax.py)
 * [Comparison with tianshou Batch](https://github.com/opendilab/treevalue/blob/main/test/compare/tianshou/test_tianshou_batch.py)
 
-## Extension
+## Change Log
 
-If you need to translate `treevalue` object to runnable source code, you may use the [potc-treevalue](https://github.com/potc-dev/potc-treevalue) plugin with the installation command below
+<details><summary><b>Version History</b> <i>[click to expand]</i></summary>
+<div>
 
-```shell
-pip install potc-treevalue
-```
+* 2022-05-03
+        1.3.1: Change definition of getitem, setitem and delitem; add pop method for TreeValue class.
+* 2022-03-15
+        1.3.0: Add getitem, setitem and delitem for adding, editing and removing items in TreeValue class.
+* 2022-02-22
+  	1.2.2: Optimize union function; add walk utility method.
+* 2022-01-26
+        1.2.1: Update tree printing; add keys, values, items on TreeValue; add comparision to facebook nest library.
+* 2022-01-04
+        1.2.0: Add flatten_values and flatten_keys; fix problem in mapping function; add support for potc.
+* 2021-12-03
+        1.1.0: Add version information; fix bug of default value; add flatten and unflatten; optimization speed performance.
+* 2021-10-24
+        1.0.0: Greatly optimize the speed performance using cython, overhead has been reduced to a negligible level.
+</div>
+</details>
 
-Or just install it with `treevalue` itself
 
-```shell
-pip install treevalue[potc]
-```
+## Join and Contribute
 
```

Comment:
[File an issue](https://github.com/opendilab/treevalue/issues/new/choose) on Github

Reply:
This line is added below.